### DISCUSSION TITLE
install nitrux

### DISCUSSION THREAD

**Discussion by Ilyha4ws** [Nitrux/nitrux-bug-tracker#161] [Q&A] [answered]
Hello everyone, when I try to install nitrux, I constantly get an error: it was not possible to change the partition. I’m trying to install with a second system, I allocated 30GB separately, maybe because I choose to install next to windows in automatic mode? And it also appears according to the time zone...
![изображение_viber_2024-02-11_18-04-03-784](https://github.com/Nitrux/nitrux-bug-tracker/assets/159352978/bebe7ff7-e424-443a-965b-2d8d8cc5efa3)
![изображение_viber_2024-02-11_18-11-42-136](https://github.com/Nitrux/nitrux-bug-tracker/assets/159352978/1540bd81-c87e-4f7a-a191-2a27c6e82465)

**Comment by UriHerrera** [comment, marked as answer]
Please follow the installation tutorial. The problem is that you're not allocating enough storage space.
